src/_data/catalog/widgets.json

```diff
@@ -1707,6 +1983,16 @@
     "link": "https://api.flutter.dev/flutter/material/TabPageSelector-class.html",
     "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-tab-bar.png'>"
   },
+  {
+    "name": "Table",
+    "description": "A widget that uses the table layout algorithm for its children.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Displays child widgets in rows and columns.",
```